op-guide/Monitoring_TiDB.md

```diff
@@ -0,0 +1,183 @@
+
+- Prometheus Server: [https://github.com/prometheus/prometheus#install](https://github.com/prometheus/prometheus#install)
+
+- Grafana: [http://docs.grafana.org](http://docs.grafana.org/)
+
+### Configuration
+
+#### Configuring TiDB, PD and TiKV
+
+- TiDB: Set the two parameters: `--metrics-addr` and `--metrics-interval`. 
+
+    - Set the Push Gateway address as the `--metrics-addr` parameter. 
+
+    - Set the push frequency as the `--metrics-interval` parameter. The unit is s, and the default value is 15.
+
+- PD: update the toml configuration file with the Push Gateway address and the the push frequency: 
+
+	```bash
+	[metric]
	\# prometheus client push interval, set "0s" to disable prometheus.
	interval = "15s"
	\# prometheus pushgateway address, leaves it empty will disable prometheus.
	address = "host:port"
+	```
+
+* TiKV: update the toml configuration file with the Push Gateway address and the the push frequency. Set the job field as "tikv"
+
+	```bash
+	[metric]
	\# the Prometheus client push interval. Setting the value to 0s stops Prometheus client from pushing.
	interval = "15s"
	\# the Prometheus pushgateway address. Leaving it empty stops Prometheus client from pushing.
	address = "host:port"
	\# the Prometheus client push job name. Note: A node id will automatically append, e.g., "tikv_1".
	job = "tikv"
+	```
+#### Configuring PushServer
+
+Generally, it does not need to be configured. You can use the default port: 9091.
+
+#### Configuring Prometheus
+
+Add the Push Gateway address to the yaml configuration file:
+
+	```bash
+	scrape_configs:
	  \# The job name is added as a label `job=<job_name>` to any timeseries scraped from this config.
	  - job_name: 'TiDB'
	
	    \# Override the global default and scrape targets from this job every 5 seconds.
	    scrape_interval: 5s
	
	    honor_labels:true
	
	    static_configs:
	      - targets: ['host:port'] # use the Push Gateway address
	        labels:
	                group: 'production'
+	```
+
+#### Configuring Grafana
+
+##### Creating a Prometheus data source
+
+1. Login the Grafana Web interface. 
+
+    * The default address is: [http://localhost:3000](http://localhost:3000)
+
+    * The default account name: admin
+
+    * The password for the default account: admin
+
+2. Click the Grafana logo to open the sidebar menu.
+
+3. Click "Data Sources" in the sidebar.
+
+4. Click "Add data source".
+
+5. Specify the data source information:
+
+    * Specify the name for the data source. 
+
+    * For Type, select Prometheus.
+
+    * For Url, specify the Prometheus address.
+
+    * Specify other fields as needed.
+
+6. Click "Add" to save the new data source.
+
+##### Creating a Grafana dashboard
+
+1. Click the Grafana logo to open the sidebar menu.
+
+2. On the sidebar menu, click "Dashboards" -> "Import" to open the "Import Dashboard" window.
+
+3. Click "Upload .json File" to upload a JSON file. 
+
+4. Click "Save & Open". 
+
+5. A Prometheus dashboard is created.
+
```

Comment:
https://github.com/pingcap/docs/pull/60/files#diff-f68c1b6c7f67810b2efd0959890e66e1R191
here need change `honor_labels:true` to `honor_labels: true`